components/alert/AlertMap.vue

```diff
@@ -327,365 +402,446 @@ export default Vue.extend({
+        if (region.faction === connection.faction) {
+          link.setStyle({
+            color: MAP_LINK_COLORS[region.faction]?.toString(),
+            opacity: MAP_LINK_COLORS[region.faction]?.a,
             dashArray: [],
-          });
-          bglink?.setStyle({
-            opacity: 0.0
-          });
+          })
+          bglink.setStyle({
+            stroke: false,
+          })
         }
         // Disabled, just put an NS link there since it doesn't make it seem capturable
-        else if(regions[0].faction === Faction.NONE || regions[1].faction === Faction.NONE){
-          link?.setStyle({
+        else if (
+          region.faction === Faction.NONE ||
+          connection.faction === Faction.NONE
+        ) {
+          link.setStyle({
             color: MAP_LINK_COLORS[Faction.NONE].toString(),
             opacity: MAP_LINK_COLORS[Faction.NONE].a,
             dashArray: [],
-          });
-          bglink?.setStyle({
-            opacity: 0.0
-          });
+          })
+          bglink.setStyle({
+            stroke: false,
+          })
         }
         // Warpgate's ain't capturable, make it look like capturable link with disabled link colors
-        else if(regions[0].facilityType === FacilityType.WARPGATE || regions[1].facilityType === FacilityType.WARPGATE){
-          link?.setStyle({
+        else if (
+          region.facilityType === FacilityType.WARPGATE ||
+          connection.facilityType === FacilityType.WARPGATE
+        ) {
+          link.setStyle({
             color: MAP_LINK_COLORS[0].toString(),
-            dashArray: "5 5"
-          });
-          bglink?.setStyle({
+            dashArray: '5 5',
+          })
+          bglink.setStyle({
+            stroke: true,
             color: MAP_LINK_COLORS[6].toString(),
             opacity: MAP_LINK_COLORS[6].a,
-            dashArray: "5 5",
-            dashOffset: "5"
-          });
+            dashArray: '5 5',
+            dashOffset: '5',
+          })
         }
         // Enemy factions, set capturable color, dashes, and enable bglink
         else {
-          link?.setStyle({
+          link.setStyle({
             color: MAP_LINK_COLORS[4].toString(),
-            dashArray: "5 5"
-          });
-          bglink?.setStyle({
+            dashArray: '5 5',
+          })
+          bglink.setStyle({
+            stroke: true,
             color: MAP_LINK_COLORS[5].toString(),
             opacity: MAP_LINK_COLORS[5].a,
-            dashArray: "5 5",
-            dashOffset: "5"
-          });
+            dashArray: '5 5',
+            dashOffset: '5',
+          })
         }
-      });
+      })
     },
     async pull(): Promise<void> {
-        if (this.loaded && this.alert.state === Ps2alertsEventState.ENDED) {
-          return
-        }
+      if (this.loaded && this.alert.state === Ps2alertsEventState.ENDED) {
+        return
+      }
+
+      this.outfitData = await this.pullOutfitData(
```

Comment:
May want to consider using a `promise.all` here, so you can get both the outfit data and the alert map data at the same time (thus saving a few ms) and once you have both, process the data.

It would look something like this:

```typescript
promise.all(
  await this.pullOutfitData(),
  await new ApiRequest(...),
).then(values) {
  this.outfitData = values[0]
  // ... the rest of code, data of which in values[1]
}
```

May not need the await, think it comes built in actually.

I also realise I may be doing the same thing in places 😅

Reply:
👍